lib/token.h

```diff
@@ -545,6 +554,15 @@ class CPPCHECKLIB Token {
     void isAttributeMaybeUnused(const bool value) {
         setFlag(fIsAttributeMaybeUnused, value);
     }
+    std::vector<std::string> getAttributeAlignas() const {
+        return !mImpl->mAttributeAlignas ? std::vector<std::string>() : *mImpl->mAttributeAlignas;
```

Comment:
No mistake here but It got me confused for a moment to read it with negation `!`. So just to make it a bit easy to read
`mImpl->mAttributeAlignas ? *mImpl->mAttributeAlignas : std::vector<std::string>();`